deployments/karnot/deployment-config.yaml

```diff
@@ -1,70 +1,56 @@
-          UNINITIALIZED_ACCOUNT_CLASS_HASH: '0x7b2de5e73ff7eb338d76c967dd5aa3f3004574d326b8c1402bb819d4983b8b6'
-          WHITE_LISTED_EIP_155_TRANSACTION_HASHES: '0xeddf9e61fb9d8f5111840daef55e5fde0041f5702856532cdbb5a02998033d26,0x07471adfe8f4ec553c1199f495be97fc8be8e0626ae307281c22534460184ed1,0xb6274b80bc7cda162df89894c7748a5cb7ba2eaa6004183c41a1837c3b072f1e'
-          MAX_LOGS: '10000'
-          RETRY_TX_INTERVAL: '5'
-
+          KAKAROT_ADDRESS: "0x1d2e513630d8120666fc6e7d52ad0c01479fd99c183baac79fff9135f46e359"
+          UNINITIALIZED_ACCOUNT_CLASS_HASH: "0x7b2de5e73ff7eb338d76c967dd5aa3f3004574d326b8c1402bb819d4983b8b6"
+          RELAYERS_ADDRESSES: "0x219c1e7585b134f94df1a7440676e1f64668cef6ae46cebacf8e12fdb7d1ed7,0x1bc0a910803c09dc59d87d3e414c179c0fb70fca556443b694a33e06f1cb836,0x4756b97b36ff56781960e37f9725e91d4e7f10eabd538a963a16cd718a5b2ac,0x3ad2e0550210d3905881c5eb02b2dc58a4bf9145b286b020e889b76e5d15cc9,0x6f8e544a82e59227712682988d4bbba180f919afae5395bb27c088cba31aac3,0x61e109bf9e9db553c12abfe4b1d4974b71df9b5f999f80731ceb9a8d397b0eb,0x4a0ee234a5a11c942a235b34a2ef09edd61a3ba5803a64455037c430647c901,0x28e72aea13153bad9dcc71be7ad3f4e7bf769fbcbc59d39cd991783c06f126d,0x5407d8d28c43c10bdb7193b2c1553ab881ca77eb2ebc24c1ab02e848455dd99,0x62daa696aa71e97169763fb92729e2ce37a9794da4076e005acf82de6b02449,0x7116732a2ca8f5423581de26a8b316b95f2fa3043e5ac969d556dc511eaf09f,0x72f5988d56d43ddff00e59bdd8298f4c429131b5362e96ed8a12262a3a1d119,0x3c68d3a57ed0e256bc8373318b92c3764d3a460580af0ce66668f0a3f5e0ffc,0x1ed83bb61c70e198ab2a835a9a7657fb7c2c5dded4e9b4de83e4f106e0d07be,0xfd54568b763ffc311f2d058d92053dd707ca8f6d7eadefed9e3f0ad689059c,0x8d57e878e1f12b875d7b4008c15e3fed3323864d7c77966a50f6fb71496314,0x79ebdd91403cea8fc4fb7c53d7a69caea33c2fc0f171a871f8bc8ed38b73a29,0x55fe9ac4db519377c1e1d6331e6a45e3a5ce75bfae7447677be6828c2130317,0x5e5a9efc076bad92061124207e9176b36740c2539c40089fde11e4d07cc90d0,0x43a8ce7a0cb8b714e1c74d19f7b5b0a43f534edc1f7dd3e455fe7eacfa2b181,0x569c7a1ff7e5ac7e18b991ce0b78a97ee7da4710367a293625d4102be782e29,0x58a5b0e3da6eaea2860c200b6977fb08e06a051f2bbc9145d84727f04fe1ab5,0x5746bdc6296ba15a9664afa01062b9ad06f1ff31ac3b70b92eb6f875e618fcb,0x86bb9ec8022d3cc567d07ea2123778a045468c63d87dfdc0c568d20648c5f5,0x30be9c4b8032950f72f14c1b3b77121572be042ddecdf73d97e50682158777a,0x4a24cf6a552591dde0f654c15c42227dbdb66d7634aee144703c61d1b519268,0x595e3ee91ac131371399044b603b5909f33d44bdf72748e5233f6394098f5ba,0xe76bd52863c4d15f8fc4b1e3c51f1ed6c0a29254f5912b2dcfab8752d32f11,0x4806466971baa2a234e49409340ee1c0a703a7865176c01dea458ae0d6a8b06,0x3a427aca2f11e22ef583e131013a81a37054df72ee48dded88c3f1a83463c16"
```

Comment:
may we can create an issue to make this config look at bit better, it's a bit painful to look at all these, 
later on it might be better to load them from JSON rather than env string

Reply:
https://github.com/kkrt-labs/kakarot-rpc/issues/1497